internal/clients/fleet/fleet.go

```diff
@@ -399,6 +387,14 @@ func Uninstall(ctx context.Context, client *Client, name, version string, force
 	switch resp.StatusCode() {
 	case http.StatusOK:
 		return nil
+	case http.StatusBadRequest:
+		msg := resp.JSON400.Message
+		// {"statusCode":400,"error":"Bad Request","message":"some-integration is not installed"}
+		if msg != nil && strings.Contains(*msg, "is not installed") {
```

Comment:
IIUC this could be this? I don't actually feel strongly here, just checking my understanding. 

```suggestion
		if msg != nil && *msg == fmt.Sprintf("%s is not installed", name) {
```

Reply:
agreed, this works better. must not have realized name was already accessible.